Automattic/vip-go-mu-plugins
# Update jetpack 9.7 subtree to the final tag

## Description
Update jetpack 9.7 subtree to the final tag


## Changelog Description

### Plugin Updated: Jetpack 9.7

We upgraded Jetpack 9.7 to the final release. This only impacts sites with pinned version to 9.7. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Pin a site by adding `define ('VIP_JETPACK_PINNED_VERSION', '9.7');` to config.
1. Navigate to jetpack section in admin and check the version on the bottom
